src/Drupal/DrupalExtension/Manager/DrupalAuthenticationManager.php

```diff
@@ -103,6 +103,7 @@ public function logIn(\stdClass $user)
     public function logout()
     {
         $this->getSession()->visit($this->locatePath($this->getDrupalText('logout_url')));
+        $this->getSession()->getPage()->findButton($this->getDrupalText('log_out'))->click();
```

Comment:
Maybe we should just check if the element exist first? So that it's compatible with versions before 10.3 and after?

Reply:
I have addressed this in an alternate solution [#681]